app/models/host.rb

```diff
@@ -143,6 +143,9 @@ class Host < ApplicationRecord
   virtual_total :v_total_vms, :vms
   virtual_total :v_total_miq_templates, :miq_templates
 
+  scope :active,   -> { where.not(:ems_id => nil) }
+  scope :archived, -> { where(:ems_id => nil) }
```

Comment:
These are handy scopes to have, only the `active` scope here is used but both are generally helpful